Source/SwiftLintFramework/Rules/Style/IdentifierNameRule.swift

```diff
@@ -34,7 +34,11 @@ struct IdentifierNameRule: ASTRule, ConfigurationProviderRule {
         }
 
         return validateName(dictionary: dictionary, kind: kind).map { name, offset in
-            guard !configuration.excluded.contains(name), let firstCharacter = name.first else {
+            guard let firstCharacter = name.first else { return [] }
+
+            guard !configuration.excludedRegularExpressions.contains(where: {
+                return !$0.matches(in: name, options: [], range: NSRange(name.startIndex..., in: name)).isEmpty
```

Comment:
This should be a method in the configuration to avoid duplications.

Reply:
That would be much better, thanks :)